CSC8503CoreClasses/GameServer.h

```diff
@@ -13,17 +13,20 @@ namespace NCL {
 			void Shutdown();
 
 			void SetGameWorld(GameWorld &g);
+			void AddPeer(int peerNumber) const;
 
 			bool SendGlobalPacket(int msgID);
 			bool SendGlobalPacket(GamePacket& packet);
 			bool SendVariableUpdatePacket(VariablePacket& packet);
+			bool GetPeer(int peerNumber, int& peerId) const;
 
 			virtual void UpdateServer();
 
 		protected:
 			int			port;
 			int			clientMax;
 			int			clientCount;
+			int*        peers;
```

Comment:
Member variables should begin with a lower-case m; so mPeers

Reply:
Done 👍